ext/intl/msgformat/msgformat_data.h

```diff
@@ -26,16 +32,22 @@ typedef struct {
 	intl_error      error;
 
 	// formatter handling
-	UMessageFormat* umsgf;
+	UMessageFormat *umsgf;
 	char*			orig_format;
 	zend_ulong		orig_format_len;
 	HashTable*		arg_types;
 	int				tz_set; /* if we've already the time zone in sub-formats */
 } msgformat_data;
 
 msgformat_data* msgformat_data_create( void );
```

Comment:
Why not include this one too? Otherwise it becomes a bit difficult to track which ones fall under `extern "C"` and which ones don't

Reply:
fair